WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController+Me.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+extension BlogDetailsViewController {
+
+    @objc func downloadGravatarImage(for row: BlogDetailsRow) {
+        guard let email = blog.account?.email else {
+            return
+        }
+
+        ImageDownloader.shared.downloadGravatarImage(with: email) { [weak self] image in
+            guard let image,
+                  let gravatarIcon = image.gravatarIcon() else {
+                return
+            }
+
+            row.image = gravatarIcon
+            self?.tableView.reloadData()  // FIXME: only reload Me row
```

Comment:
Will it be addressed in the scope of this PR?

Reply:
Yes! https://github.com/wordpress-mobile/WordPress-iOS/pull/21348/commits/f4db05433065e6ec9fc11455ee39143e1938dc31